src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx

```diff
@@ -0,0 +1,105 @@
+
+### Step 2: Install the MySQL agent [#mysql-install]
+
+Our LAMPy integration relies on the MySQL agent to work. To learn more and check requirements review our [MySQL docs](https://docs.newrelic.com/install/mysql/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Infrastructure & OS  > MySQL**.
+2. Follow the instructions to install the MySQL agent.
+
+### Step 3: Install the Apache agent [#apache-install]
+
+Our LAMPy integration relies on the Apache agent to work. To learn more and check requirements review our [Apache docs](/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Infrastructure & OS  > Apache**.
+2. Follow the instructions on the screen to install Apache agent.
+
+### Step 4: Install the Python agent [#python-install]
+
+Our LAMPy integration relies on the Python agent to work. To learn more and check requirements review our [Python docs](/docs/apm/agents/python-agent/getting-started/introduction-new-relic-python/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Application monitoring  > Python**.
+2. Name your application.
+3. Download the configuration file and place it in your application's root directory.
+4. Integrate python agent for Django web site which is running on Apache server 
+5. Update the wsgi.py and settings.py files `settings.py` `NEW_RELIC_CONFIG_FILE = BASE_DIR / `downloaded_newrelic_file`
+6. Add the lines below to your `wsgi.py`
+
+```
+      import newrelic.agent
+
+      from django.conf import settings 
+      from django.core.wsgi import get_wsgi_application
+
+      application = get_wsgi_application()
+
+      newrelic.agent.initialize(settings.NEW_RELIC_CONFIG_FILE)
+      newrelic.agent.WSGIApplicationWrapper(application)
+```
+8. Restart your Apache server. 
+
+Wait five minutes and then proceed to finding your data in New Relic. 
+
+
+## Find your data [#find-data]
+
+To get your LAMPy dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click on **Dashboards**.
+3. In the search bar, type `LAMPy`. 
+4. The LAMPy dashboard should appear. Click on it to install it.
+
+Your LAMPy dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+This integration reports data in the form of our [browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). If you installed the infrastructure agent, you'll also receive [infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
```

Comment:
this needs review. in this case, I think it will be reporting data from the infrastructure agent and the MySQL and Apache integrations, so maybe we just link to those docs? I'm not sure if it reports a smaller subset of that data but maybe see if the SMEs think it's okay to say 'This reports data from all these places' and just link to those docs.

Reply:
@pkosanam what do you think about @zuluecho9's comment: "This reports data from all these places' and just link to those docs."

Reply:
Hi @homelessbirds , As we have mentioned the configuration steps in the above paragraphs, I feel giving the documentation links here should be fine. 👍

Reply:
thanks @pkosanam !